src/core/services/interimElement/interimElement.js

```diff
@@ -386,7 +386,7 @@ function InterimElementProvider() {
 
               function startHideTimeout() {
                 if (options.hideDelay) {
-                  hideTimeout = $timeout(service.cancel, options.hideDelay) ;
```

Comment:
@rschmukler - this changes seems reasonable: hide === close. 
What is the difference (wrt `interimElement` between `cancel()` and `hide()`).

btw, a promise wrapper is also needed in hide() change is also needed:

```js
function hide(response) {
  var interimElement = stack.shift();
  return $q.when(interimElement && interimElement.remove().then(function() {
    interimElement.deferred.resolve(response);
  }));
}
```